tests/gateway/test_braintree.py

```diff
@@ -205,7 +205,7 @@ def test_get_client_token(mock_gateway, gateway_config):
     client_token = "client-token"
     mock_generate = Mock(return_value="client-token")
     mock_gateway.return_value = Mock(client_token=Mock(generate=mock_generate))
-    result = get_client_token(gateway_config.connection_params)
+    result = get_client_token(gateway_config)
```

Comment:
So because you are here...  Maybe you can name it to something like `token` and `expected_token`? 
And maybe you can check if these mocks don't do so many magic and we don't test our logic only for testing
Or you can simply merge it